docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/upgrade-guides/2-0-0-upgrade-guide/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "2.0.0 upgrade guide"
+sidebar_position: -20
+---
+
+## Configuration
+
+BigQuery Loader 2.0.0 brings changes to the loading setup. It is no longer neccessary to configure and deploy three (in [1.X](/docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/previous-versions/bigquery-loader-1.x/index.md) it was loader, repeater and mutator) independent applications in order to load your data to BigQuery.
+Starting from 2.0.0 only one appliction is needed, which naturally introduces some breaking changes to the configuration file structure.
+
+See the [configuration reference](/docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/configuration-reference/index.md) for all possible configuration parameters
+and the minimal [configuration samples](https://github.com/snowplow-incubator/snowplow-bigquery-loader/blob/v2/config) for each of supported cloud environments.
+
+## Events table format
+
+Starting from 2.0.0, BigQuery Loader no longer uses full schema version in column names for self-describing events and entities in `events` table. It uses only major schema version in the column name instead.
```

Comment:
I suggest drop in an example here.  Pick a commonly used schema, and say what the column is called before and after the upgrade.  Will be much quicker for the user to grasp, rather than explaining it with words.

After that, I think you can condense the first two paragraphs into fewer words.